### PR TITLE
Add maintenance function to remove old import tasks

### PR DESCRIPTION
**Pull request by jamesrkiger**
## Checklist

1. [ ] If you've added code that should be tested, add tests
2. [ ] If you've changed APIs, update (or create!) the documentation
3. [x] Ensure the tests pass
4. [x] Make sure that your code lints and that you've followed [our coding style](https://github.com/kobotoolbox/kpi/blob/master/CONTRIBUTING.md)
5. [x] Write a title and, if necessary, a description of your work suitable for publishing in our [release notes](https://community.kobotoolbox.org/tag/release-notes)
6. [ ] Mention any related issues in this repository (as #ISSUE) and in other repositories (as kobotoolbox/other#ISSUE)
7. [ ] Open an issue in the [docs](https://github.com/kobotoolbox/docs/issues/new) if there are UI/UX changes

## Description

Adds maintenance task for cleaning up old import tasks. 

## Notes

I created a new test file for this. All other import task related tests are run in API test files and this didn't seem relevant to API testing. 

## Related issues

implements work from #2434

### CODE REVIEW

File: kpi/maintenance_tasks.py
Comment (bufke) [inline kpi/maintenance_tasks.py:33]: @noliveleger in the original PR you wrote "(90 by default)" is there any reason to make this configurable? I like hard coding it, if not. Would a private server ever say want to make it longer?

File: kpi/maintenance_tasks.py
Comment (bufke) [inline kpi/maintenance_tasks.py:37]: Now that we're using this a few times, we could consider adding abstraction. I'm fine doing this right now or in the next batch of work. 

`chunked_delete(queryset: QuerySet)`

File: kpi/tests/test_import_tasks.py
Comment (bufke) [inline kpi/tests/test_import_tasks.py:24]: I assume we couldn't set date_created directly and this works around it. Does it work when using model bakery?

**Review by bufke — APPROVED** [review]
LGTM pending the review of my comments

File: kpi/maintenance_tasks.py
Comment (jamesrkiger) [reply kpi/maintenance_tasks.py:37]: @bufke This will actually be the last PR for the KPI-portion of the Celery maintenance work. Should I just abstract this now or should we hold off until some theoretical future maintenance task tips it over into definite DRY territory?

File: kpi/tests/test_import_tasks.py
Comment (jamesrkiger) [reply kpi/tests/test_import_tasks.py:24]: Correct, we can't set it on creation. Doesn't work with model bakery, either. I have added a note

File: kpi/maintenance_tasks.py
Comment (noliveleger) [reply kpi/maintenance_tasks.py:29]: It's part of an old discussion of the back-end team. IIRC, yes some private servers wanted a different value. So please keep it configurable.

File: kpi/tests/test_import_tasks.py
Comment (noliveleger) [inline kpi/tests/test_import_tasks.py:4]: Please don't import `User` directly.  

See https://chat.kobotoolbox.org/#narrow/stream/55-KoboCAT-Replacement/topic/KoboCAT.20as.20Django.20app/near/334486

**Review by noliveleger — CHANGES_REQUESTED** [review]

File: kpi/maintenance_tasks.py
Comment (jamesrkiger) [reply kpi/maintenance_tasks.py]: Added constance var

**Pull request merged**
